extensions/amp-user-location/validator-amp-user-location.protoascii

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-user-location
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-user-location"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-user-location>
+  html_format: AMP
+  tag_name: "AMP-USER-LOCATION"
+  requires_extension: "amp-user-location"
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-user-location"
```

Comment:
These are being moved to amp.dev and since this is a new file it won't be included in the move. So feel free to have it as amp.dev now.

```
spec_url: "https://amp.dev/documentation/components/amp-user-location"
```